constants/constants.go

```diff
@@ -12,3 +12,5 @@ const (
 	SectionUSPUT  SectionID = 11
 	SectionUSPCT  SectionID = 12
 )
+
+const SectionGPPByte byte = 'D'
```

Comment:
Not sure if here or where it's used is better, but could you add a comment explaining how this is used?

Reply:
I moved this and added comments.